databricks/koalas/generic.py

```diff
@@ -1300,10 +1339,12 @@ def min(self, axis=None, numeric_only=None) -> Union[Scalar, "Series"]:
         1.0
         """
         return self._reduce_for_stat_function(
-            F.min, name="min", numeric_only=numeric_only, axis=axis
+            F.min, name="min", axis=axis, numeric_only=numeric_only
         )
 
-    def max(self, axis=None, numeric_only=None) -> Union[Scalar, "Series"]:
+    def max(
+        self, axis: Union[int, str] = None, numeric_only: bool = None
```

Comment:
Why do we use `numeric_only: bool = None` rather than `numeric_only: bool = False`?

Reply:
Yes, I think so. cc @itholic who changed it to `None`.

Reply:
Yes, It's for pandas compatibility.

Reply:
Thanks!